docs/deployment_guide/partner_editable/taskcat_configuration.adoc

```diff
@@ -0,0 +1,55 @@
+// Edit this placeholder text as necessary to describe the deployment options.
+
+We will use https://aws-ia.github.io/taskcat/[taskcat] tool to package and upload the code artifacts. Taskcat is a tool that will build and bundle the CFN template into an S3 Bucket which users can then use to spin up their cloudformation stack.
+
+The configuration for the taskcat tool is present in the `.taskcat.yml` in the root of the project directory. 
+
+** Example `.taskcat.yml` configuration
++
+[source, yaml]
+----
+project:
+  name: cfn-ecr-aws-soci-index-builder
+  owner: quickstart@amazon.com
+  s3_regional_buckets: true
+  lambda_source_path: functions/source
+  lambda_zip_path: functions/packages
+  parameters:
+    QSS3BucketName: $[taskcat_autobucket]
+    QSS3KeyPrefix: "cfn-ecr-aws-soci-index-builder/"
+tests:
+  can-deploy:
+    parameters:
```

Comment:
This needs `SociVersion: "V2"`, doesn't it?

Reply:
The `project` doesn't need the SociVersion param, the `tests` does and it has been added